STEERING-COMMITTEE.md

```diff
@@ -62,10 +61,8 @@ KSC meets every two weeks, or as-needed. Meetings are held online.
 
 Given the private nature of many of these discussions (e.g. privacy, private
 emails to the committee, code of conduct violations, escalations, disputes
-between members, security reports, etc.) meetings are held in private.
-
-Meeting notes are available to members of the knative-dev mailing list (link to
-be added).
+between members, security reports, etc.) most meetings are held in private. 
+However, a Steering meeting open to the public is held on the 3rd Wednesday of each month, and all are welcome to attend.
```

Comment:
Will there be meeting notes + recordings for folks who can't attend? I can totes see not necessarily having all the notes visible (see reference to private meetings), but I don't see there being a downside for having notes and recordings for the public meetings?

Reply:
I don't want to speak for the whole committee; but I think recordings make sense!